tools/replica/reorder_remd_traj.py

```diff
@@ -0,0 +1,571 @@
+#!/usr/bin/env python
+
+"""
+LAMMPS Replica Exchange Molecular Dynamics (REMD) trajectories are arranged by
+replica, i.e., each trajectory is a continuous replica that records all the 
+ups and downs in temperature. However, often the requirement is trajectories
+that are continuous in temperature, which is achieved by this tool.
+
+Author: 
+Tanmoy Sanyal, Shell lab, Chemical Engineering, UC Santa Barbara
+Email: tanmoy dot 7989 at gmail dot com
+
+Usage
+-----
+To get detailed information about the arguments, flags, etc use:
+python reorder_remd_traj.py -h or 
+python reorder_remd_traj.py --help
+
+Features of this script
+-----------------------
+a) reorder LAMMPS REMD trajectories by temperature keeping only desired frames.
+Note: this only handles LAMMPS format trajectories (i.e. .lammpstrj format)
+Trajectories can be gzipped or bz2-compressed. The trajectories are assumed to
+be named as <prefix>.%d.lammpstrj[.gz or .bz2]
+
+b) (optionally) calculate configurational weights for each frame at each
+temperature if potential energies are supplied. But this if for the canonical
+(NVT) ensemble only.
+
+Dependencies
+------------
+mpi4py
+pymbar (for getting configurational weights)
+tqdm (for printing pretty progress bars)
+StringIO (or io if in Python 3.x)
+
+"""
+
+
+
+import os, sys, numpy as np, argparse, time, pickle
+from scipy.special import logsumexp
+from mpi4py import MPI
+
+from tqdm import tqdm, trange
```

Comment:
it seems like you do not use the sys module and the trange function from tqdm.

Reply:
removed these imports